auth-service/src/main/kotlin/uk/gov/communities/delta/auth/services/RegistrationService.kt

```diff
@@ -73,22 +73,23 @@ class RegistrationService(
     }
 
     private suspend fun addUserToOrganisations(adUser: UserService.ADUser, organisations: List<Organisation>) {
-        logger.atInfo().addKeyValue("UserDN", adUser.dn).log("User added to domain organisations")
+        logger.atInfo().addKeyValue("UserDN", adUser.dn).log("Adding user to domain organisations")
         try {
             organisations.forEach {
                 if (!it.retired) {
                     groupService.addUserToGroup(
                         adUser,
                         organisationUserGroup(it.code)
                     )
-                    logger.atInfo().addKeyValue("UserDN", adUser.dn)
-                        .log("Added user to domain organisation with code {}", it.code)
```

Comment:
Do we not need to know which organisations specifically the user is added to for audit purposes?